tests/integration_tests/clouds.py

```diff
@@ -25,6 +26,65 @@
 log = logging.getLogger('integration_testing')
 
 
+def _get_ubuntu_series() -> list:
+    """Use distro-info-data's ubuntu.csv to get a list of Ubuntu series"""
+    try:
+        with open("/usr/share/distro-info/ubuntu.csv") as f:
```

Comment:
Why not avoid the csv import and use `distro-info -a` directly to simlify parsing?

Reply:
Done, but with `ubuntu-distro-info` (else we'd guess Debian releases as "ubuntu:<release>" when the tests run on a Debian machine).